scripts/generate.sh

```diff
@@ -67,3 +67,16 @@ mv -f src/README.md ./README.md
 mv -f src/composer.json composer.json
 find ./docs -type f ! -name "DecryptionKey.md" ! -name "Sealed.md" -exec rm {} +
 mv -f src/docs/* ./docs
+
+declare -a modelsWithRawResponse=("./src/Model/EventResponse.php" "./src/Model/Response.php")
+
+for model in "${modelsWithRawResponse[@]}"; do
+  # enable WithRawResponse trait for the model
+  (
+    if [ "$platform" = "Darwin" ]; then
+      sed -i '' 's/\/\/ use \\/use \\/' $model
+    else
+      sed -i 's/\/\/ use \\/use \\/' $model
+    fi
+  )
+done
```

Comment:
Could you explain why we need this transformation?

Reply:
To enable the use of raw response trait, drop the comment as you asked in the previous message